subsys/nrf_security/src/drivers/Kconfig.psa_accel

```diff
@@ -0,0 +1,821 @@
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB224
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB256
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB320
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB384
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB512
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R163
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R192
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R233
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R283
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R409
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R521
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R571
+	bool
+
+config PSA_ACCEL_KEY_PAIR_X25519
+	bool
+
+config PSA_ACCEL_MD5
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA1
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA224
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA256
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA384
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA512
+	bool
+
+config PSA_ACCEL_RANDOM
+	bool
+
+config PSA_ACCEL_RIPEMD160
+	bool
+
+config PSA_ACCEL_RSA1024_PKCS1V15_CRYPT
+	bool
+
+config PSA_ACCEL_RSA1024_SHA1_OAEP
+	bool
+
+config PSA_ACCEL_RSA1024_SHA1_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1024_SHA1_PSS
+	bool
+
+config PSA_ACCEL_RSA1024_SHA224_OAEP
+	bool
+
+config PSA_ACCEL_RSA1024_SHA224_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1024_SHA224_PSS
+	bool
+
+config PSA_ACCEL_RSA1024_SHA256_OAEP
+	bool
+
+config PSA_ACCEL_RSA1024_SHA256_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1024_SHA256_PSS
+	bool
+
+config PSA_ACCEL_RSA1024_SHA384_OAEP
+	bool
+
+config PSA_ACCEL_RSA1024_SHA384_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1024_SHA384_PSS
+	bool
+
+config PSA_ACCEL_RSA1024_SHA512_OAEP
+	bool
+
+config PSA_ACCEL_RSA1024_SHA512_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1024_SHA512_PSS
+	bool
+
+config PSA_ACCEL_RSA1536_PKCS1V15_CRYPT
+	bool
+
+config PSA_ACCEL_RSA1536_SHA1_OAEP
+	bool
+
+config PSA_ACCEL_RSA1536_SHA1_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1536_SHA1_PSS
+	bool
+
+config PSA_ACCEL_RSA1536_SHA224_OAEP
+	bool
+
+config PSA_ACCEL_RSA1536_SHA224_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1536_SHA224_PSS
+	bool
+
+config PSA_ACCEL_RSA1536_SHA256_OAEP
+	bool
+
+config PSA_ACCEL_RSA1536_SHA256_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1536_SHA256_PSS
+	bool
+
+config PSA_ACCEL_RSA1536_SHA384_OAEP
+	bool
+
+config PSA_ACCEL_RSA1536_SHA384_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1536_SHA384_PSS
+	bool
+
+config PSA_ACCEL_RSA1536_SHA512_OAEP
+	bool
+
+config PSA_ACCEL_RSA1536_SHA512_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA1536_SHA512_PSS
+	bool
+
+config PSA_ACCEL_RSA2048_PKCS1V15_CRYPT
+	bool
+
+config PSA_ACCEL_RSA2048_SHA1_OAEP
+	bool
+
+config PSA_ACCEL_RSA2048_SHA1_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA2048_SHA1_PSS
+	bool
+
+config PSA_ACCEL_RSA2048_SHA224_OAEP
+	bool
+
+config PSA_ACCEL_RSA2048_SHA224_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA2048_SHA224_PSS
+	bool
+
+config PSA_ACCEL_RSA2048_SHA256_OAEP
+	bool
+
+config PSA_ACCEL_RSA2048_SHA256_PKCS1V15_SIGN
+	bool
+
+config PSA_ACCEL_RSA2048_SHA256_PSS
+	bool
+
+config PSA_ACCEL_RSA2048_SHA384_OAEP
```

Comment:
Use root from [PSA_ALG_RSA_OAEP](https://armmbed.github.io/mbed-crypto/html/api/ops/pke.html?highlight=rsa_oaep#c.PSA_ALG_RSA_OAEP)

Move key-size to the end

i.e.: PSA_ACCEL_RSA_OAEP_RSA2048_SHA_384

Reply:
Same as above, these come from the Oberon PSA core.